context/app/static/js/components/Detail/ProvVis/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import ProvVis from './ProvVis';
+
+function renderProvVis(id, prov, props = {}) {
+  const element = document.getElementById(id);
+  const { getNameForActivity, getNameForEntity, renderDetailPane } = props;
+  ReactDOM.render(
+    <ProvVis
+      prov={prov}
+      getNameForActivity={getNameForActivity}
+      getNameForEntity={getNameForEntity}
+      renderDetailPane={renderDetailPane}
+    />,
+    element,
+  );
+}
```

Comment:
Do we still need to use `ReactDOM.render`? Could we refactor to make this more declarative?

Reply:
I'll make a branch off of this PR, but I wanted to just replicate the existing behavior for now, in case doing it the right way turns out to be hard for some reason.